recipe/conda_build_config.yaml

```diff
@@ -1,11 +1,11 @@
-# match vsyear / vsver / cl_version (for CHOST suffix)
+# match vsyear / vsver / cl_version
 # from https://github.com/conda-forge/vc-feedstock/blob/main/recipe/conda_build_config.yaml
 VSYEAR:
   - 2017
 VSVER:
   - 15
-CHOST:
-  - x86_64-pc-windows-msvc19.14.26423
+CL_VERSION:
+  - 19.16.27033
```

Comment:
From what I can tell, these two versions should have been consistent between `CHOST` and the `vs2017_win-64 19.16.27033` run-req in `meta.yaml`; this commit enforces that.